plugins/modules/ilo_redfish_info.py

```diff
@@ -48,68 +49,110 @@
     description:
       - Security token for authenticating to iLO.
     type: str
+    version_added: 2.3.0
   timeout:
     description:
-      - Timeout in seconds for HTTP requests to iLO.
-    default: 10
+      - Timeout in seconds for URL requests to iLO.
+      - The default value for this param is C(10) but that is being deprecated
+        and it will be replaced with C(60) in community.general 8.0.0.
     type: int
+  cert_file:
+    description:
+      - absolute path to the server cert file
+    type: str
+    version_added: 6.1.0
+  key_file:
+    description:
+      - absolute path to the server key file
+    type: str
+    version_added: 6.1.0
 author:
-    - "Bhavya B (@bhavya06)"
+  - Bhavya B (@bhavya06)
+  - Gayathiri Devi Ramasamy (@Gayathirideviramasamy)
+  - T S Kushal (@TSKushal)
+  - Varni H P (@varini-hp)
+  - Prativa Nayak (@prativa-n)
 '''
 
 EXAMPLES = '''
   - name: Get iLO Sessions
     community.general.ilo_redfish_info:
       category: Sessions
       command: GetiLOSessions
-      baseuri: "{{ baseuri }}"
-      username: "{{ username }}"
-      password: "{{ password }}"
-    register: result_sessions
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+
+  - name: Get physical drive details
+    community.general.ilo_redfish_info:
+      category: Systems
+      command: GetSmartStoragePhysicalDrives
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+
+  - name: Get logical drive details
+    community.general.ilo_redfish_info:
+      category: Systems
+      command: GetSmartStorageLogicalDrives
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+
+  - name: Get network boot settings
+    community.general.ilo_redfish_info:
+      category: Systems
+      command: GetBootSettings
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+
+  - name: Get logical drive details with array controllers
+    community.general.ilo_redfish_info:
+      category: Systems
+      command: GetSmartStorageLogicalDrivesWithArrayControllers
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
 '''
 
 RETURN = '''
 ilo_redfish_info:
-    description: Returns iLO sessions.
+    description: Returns the retrieved information from the iLO.
     type: dict
     contains:
-        GetiLOSessions:
-            description: Returns the iLO session msg and whether the function executed successfully.
+        command:
```

Comment:
What about the existing return value?

Reply:
Since the functionality is now extended we added a generic return value which would apply to the old and new modules.